projects/packages/jetpack-mu-wpcom/tests/php/features/launchpad/class-launchpad-task-lists-test.php

```diff
@@ -434,9 +434,9 @@ function ( $extended_task_definitions ) use ( $task ) {
 	 */
 	public function provide_get_calypso_path_validation_test_cases() {
 		return array(
-			'Full URL is rejected'                       => array(
-				'https://example.com/invalid-full-url',
-				null,
+			'Full URL should be valid'                   => array(
+				'http://example.org/wp-admin/admin.php?page=jetpack#/discussion',
+				'http://example.org/wp-admin/admin.php?page=jetpack#/discussion',
```

Comment:
I think we should keep the existing test and add a separate, better-named test for admin URLs.
```suggestion
			'External absolute URL should be invalid'                   => array(
				'https://example.com/invalid-full-url',
				null,
			),
			'Admin URL should be valid'                   => array(
				'http://example.org/wp-admin/admin.php?page=jetpack#/discussion',
				'http://example.org/wp-admin/admin.php?page=jetpack#/discussion',
```